website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -450,19 +564,24 @@ UpstreamConfig = {
         {
           name: 'OutboundListenerPort',
           type: 'int: "15001"',
-          description: `The port the proxy should listen on for outbound traffic. This must be the port where 
+          description: `The port the proxy should listen on for outbound traffic. This must be the port where
                         outbound application traffic is redirected to.`,
+          yaml: false,
         },
         {
           name: 'DialedDirectly',
           type: 'bool: false',
-          description: `Determines whether this proxy instance's IP address can be dialed directly by transparent proxies. 
-                        Typically transparent proxies dial upstreams using the "virtual" tagged address, which load balances 
-                        across instances. Dialing individual instances can be helpful in cases like stateful services such 
-                        as a database cluster with a leader. `,
+          description: {
+            hcl: `Determines whether this proxy instance's IP address can be dialed directly by transparent proxies.
+              Typically transparent proxies dial upstreams using the "virtual" tagged address, which load balances
+              across instances. Dialing individual instances can be helpful in cases like stateful services such
```

Comment:
would it make sense to say "ClusterIP" to give some context here, I'm fumbling for the wording but in k8s context it's dialing the pod IP vs the clusterIP of the service?

Reply:
I do have that for the k8s docs

> Determines whether the Pod IPs can be dialed directly (versus the Cluster IP).
              Dialing Pod IPs can be helpful in cases like stateful services such
              as a database cluster with a leader or with an ingress controller that dials Pod IPs instead of Cluster IPs.

Not sure how to make this more clear for VMs. I thought as written previously it was pretty good. Open to any suggestions though.

Reply:
Good point, I do like how it was. I think I tagged the wrong line here somehow, but going back and re-reading it looks great!